src/data/cosmos_chains.json

```diff
@@ -224,6 +218,12 @@
       "network_id": "fetch",
       "image": "/images/chains/fetch.svg"
     },
+    {
+      "id": "haqq",
+      "name": "Haqq",
+      "network_id": "haqq",
+      "image": "/images/chains/haqq.svg"
```

Comment:
Image is missing:
![image](https://github.com/axelarnetwork/axelar-docs/assets/3818101/9f79234e-6622-4f07-b5f2-457c263bbf74)

Reply:
@kalidax